txmongo/collection.py

```diff
@@ -297,12 +297,11 @@ def _get_write_concern(self, safe=None, **wc_options):
                 # In this case safe=True must issue getLastError without args
                 # even if connection-level write concern was unacknowledged
                 return WriteConcern()
-        elif safe == False:
+        elif not safe:
```

Comment:
may be just `else` here? and remove final `return self.write_concern`. It is unreachable with this `elif not safe` anyways